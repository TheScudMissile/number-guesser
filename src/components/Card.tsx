import React, { ReactNode } from 'react';
import styles from '../styles/Card.style';
import { View } from 'react-native';

type CardProps = {
  children: ReactNode;
  rest?: object;
};

export default ({ children, rest }: CardProps) => (
  <View style={{ ...styles.card, ...rest }}>{children}</View>
);
