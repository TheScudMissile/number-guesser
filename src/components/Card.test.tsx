import React from 'react';
import renderer from 'react-test-renderer';
import Card from './Card';

describe('Card', () => {
  test('should render correctly', () => {
    const tree = renderer.create(<Card>test</Card>);
    expect(tree).toMatchSnapshot();
  });
});
