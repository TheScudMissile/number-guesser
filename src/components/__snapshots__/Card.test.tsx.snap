// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should render correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "elevation": 10,
      "marginTop": 100,
      "maxWidth": "80%",
      "padding": 15,
      "shadowColor": "black",
      "shadowOffset": Object {
        "height": 2,
        "width": 0,
      },
      "shadowOpacity": 0.26,
      "shadowRadius": 6,
      "width": 300,
    }
  }
>
  test
</View>
`;
